sim/battle.ts

```diff
@@ -2304,8 +2334,9 @@ export class Battle {
 	}
 
 	getTargetLoc(target: Pokemon, source: Pokemon) {
-		const position = target.position + 1;
-		return (target.side === source.side) ? -position : position;
+		let position = target.position + 1;
+		if (target.side.n % 2 === 0) position += 1;
```

Comment:
(Not sure what this is trying to do.)

Reply:
Trying to get the correct target loc. should add a check for multis tho.